<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoCalibrateDialog</class>
 <widget class="QDialog" name="AutoCalibrateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>369</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Automatic calibration requires the program to acquire a reasonable amount of samples from the channel. Configure the channel and sample skip at which to perform acquisition in the main menu. In this dialog set the acquisition duration. If the internal RAM overflows this time will be cut short.
Once the acqusistion finishes it will calculate the signal RMS and AVG values. The average value can be applied as offset for either the analog DC bias or the digital offset.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Channel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="autoCalibrateChannel">
       <property name="text">
        <string>CH1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sample skip</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="autoCalibrateSampleSkip">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Duration</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="autoCalibrateDuration">
       <property name="suffix">
        <string> ms</string>
       </property>
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
       <property name="singleStep">
        <number>500</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Offset mode</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="autoCalibrateAnalog">
         <property name="text">
          <string>Analog</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="autoCalibrateDigital">
         <property name="text">
          <string>Digital</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="autoCalibrateFine">
         <property name="text">
          <string>Fine Digital</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>AVG</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="autoCalibrateAverage">
         <property name="text">
          <string>- (- mV)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>RMS</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="autoCalibrateRMS">
       <property name="text">
        <string>- (- mV)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="autoCalibrationStatus">
       <property name="text">
        <string>OVERFLOWED</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QProgressBar" name="autoCalibrateProgress">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="autoCalibrateButton">
         <property name="text">
          <string>Run acquisition</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="autoCalibrateApplyOffset">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Apply offset</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
